tests/shell/ShellCheckBearTest.py

```diff
@@ -17,7 +17,27 @@
 echo $X
 """
 
+valid_with_ignore_rules_file = """
+#!/usr/bin/env sh
+echo "Path is $PATH"
+cd /home
+for dir in */
+do
+  cd "$dir"
+  convert index.png index.jpg
+  cd ..
+done
+"""
+
+ShellCheckBear1Test = verify_local_bear(
+    ShellCheckBear,
+    valid_files=(valid_file,),
+    invalid_files=(invalid_file, valid_with_ignore_rules_file))
 
-ShellCheckBearTest = verify_local_bear(ShellCheckBear,
-                                       valid_files=(valid_file,),
-                                       invalid_files=(invalid_file,))
+# Check that ignore_rules option working
+ShellCheckBear2Test = verify_local_bear(
+    ShellCheckBear,
+    valid_files=(valid_with_ignore_rules_file,),
+    invalid_files=(),
+    settings={
+        'shellcheck_ignore': ['SC2103', 'SC2164']})
```

Comment:
The dict will fit on one line, and the settings can have a trailing `,`, and the final `)` on the final line by itself. i.e.
```py
    settings={...},
)
```
Then, additional parameters can be added at the end of the function call without your lines being modified.